packages/graphql/src/mutations/_txHelper.js

```diff
@@ -46,7 +50,11 @@ function useRelayer({ mutation, value }) {
   if (!contracts.config.relayer) reason = 'relayer not configured'
   if (!mutation) reason = 'no mutation specified'
 
-  if (mutation === 'makeOffer' && value && value !== '0') {
+  if (
+    ['makeOffer', 'swapAndMakeOffer'].includes(mutation) &&
```

Comment:
So prior to this we were using the relayer for swapAndMakeOffer ? That's odd, I don't remember my offers with DAI payment going thru the relayer...

Reply:
No, it just so happened that it was deflected by `useProxy`.  `useProxy` didn't work for my logic so i added it here too.